app/assets/javascripts/application.js

```diff
@@ -12,12 +12,16 @@
 //
 //= require jquery
 //= require jquery_ujs
-//= require jquery.nestable
 
 // Required by Blacklight
 //= require jquery-ui
-//= require jquery.ui.nestedSortable
 //= require blacklight/blacklight
 //= require browse_everything
-//= require pop_help
-//= require input_form_attribute_support
+//= require mediaelement_rails
```

Comment:
Is this including more than we want since before we were only including mediaelement-and-player?